persistence/order.md

```diff
@@ -11,25 +11,25 @@ redirects:
 - nservicebus/persistence/order
```

Comment:
Would you also want to add a review date?

Reply:
👍